src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/Example1.java

```diff
@@ -5,7 +5,7 @@
   </module>
 </module>
 */
-// non-compiled with javac: Compilable with Java11
+// non-compiled with javac: Compilable with java17
```

Comment:
Should be `non-compiled with javac: Compilable with Java17`

Capital J.

Reply:
It should be caught by
https://github.com/checkstyle/checkstyle/blob/a3574ac91b3d06e3bb1880cccd21ea0d6c39d9d7/.ci/validation.sh#L516-L524

Reply:
Ok, now we need to exclude list in grep with such problematic examples that need very special imports.